make.js

```diff
@@ -1,324 +1,390 @@
-            + tabbing + "}\n"
-            + tabbing + "else\n"
-            + tabbing + "{\n"
-            + tabbing + "    if (container.errorCallback != nullptr)\n"
-            + tabbing + "    {\n"
-            + tabbing + "         PlayFabError error;\n"
-            + tabbing + "         error.ErrorCode = PlayFabErrorCode::PlayFabErrorEntityTokenMissing;\n"
-            + tabbing + "         error.ErrorMessage = \"The Login Attempt returned a null EntityToken. This was a mistake. Please try the login again in a moment.\";\n"
-            + tabbing + "         container.errorCallback(error, container.GetCustomData());\n"
-            + tabbing + "         return;\n"
-            + tabbing + "    }\n"
-            + tabbing + "}\n";
-    if (apiCall.result === "RegisterPlayFabUserResult")
-        return tabbing + "context->HandlePlayFabLogin(outResult.PlayFabId, outResult.SessionTicket, outResult.EntityToken->Entity->Id, outResult.EntityToken->Entity->Type, outResult.EntityToken->EntityToken);\n"
-
-    return "";
-}
-
-function ifHasProps(datatype, displayText) {
-    if (datatype.properties.length === 0)
-        return "";
-    return displayText;
-}
-
-function getVerticalNameDefault() {
-    if (sdkGlobals.verticalName) {
-        return sdkGlobals.verticalName;
-    }
-    return "";
-}
+var path = require("path");
+
+// Making resharper less noisy - These are defined in Generate.js
+if (typeof getCompiledTemplate === "undefined") getCompiledTemplate = function () { };
+if (typeof templatizeTree === "undefined") templatizeTree = function () { };
+
+exports.makeCombinedAPI = function (apis, sourceDir, apiOutputDir) {
+    console.log("Generating Combined api from: " + sourceDir + " to: " + apiOutputDir);
+
+    var removeStatic = ""; // "DISABLE_PLAYFAB_STATIC_API;";
+    var libDefines = "ENABLE_PLAYFABADMIN_API;ENABLE_PLAYFABSERVER_API;" + removeStatic;
+    var clientDefines = "" + removeStatic;
+    var serverDefines = "ENABLE_PLAYFABADMIN_API;ENABLE_PLAYFABSERVER_API;DISABLE_PLAYFABCLIENT_API;" + removeStatic;
+    var azureDefines = "DISABLE_PLAYFABCLIENT_API;";
+
+    var locals = {
+        apis: apis,
+        buildIdentifier: sdkGlobals.buildIdentifier,
+        clientDefines: clientDefines,
+        libDefines: libDefines,
+        serverDefines: serverDefines,
+        azureSdk: false,
+        azureDefines: "",
+        sdkVersion: sdkGlobals.sdkVersion,
+        sdkDate: sdkGlobals.sdkVersion.split(".")[2],
+        sdkYear: sdkGlobals.sdkVersion.split(".")[2].substr(0, 2),
+        vsVer: "v141", // As C++ versions change, we may need to update this
+        vsYear: "2017", // As VS versions change, we may need to update this
+        getVerticalNameDefault: getVerticalNameDefault,
+        winSdkVersion: "10.0.17763.0" // Which version of the Windows SDK (A VS installation option) to use
+    };
+
+    if(sdkGlobals.buildFlags.includes("azure")){

```

Comment:
This check is duplicated down below in isAzureSDK. Maybe just turn this check into a call to that method. So, something like:

if (isAzureSdk()){
...
}